components/resources/wallet_strings.grdp

```diff
@@ -114,6 +114,17 @@
   <message name="IDS_BRAVE_WALLET_OWNED" desc="Token list owned label">Owned</message>
   <message name="IDS_BRAVE_WALLET_NOT_OWNED" desc="Token list not owned label">Not owned</message>
   <message name="IDS_BRAVE_WALLET_AMOUNT_24H" desc="Token list amount and 24h change">Amount/24h</message>
+  <message name="IDS_BRAVE_WALLET_CHOOSE_ASSET_TO_BRIDGE" desc="Bridge asset modal title">Choose asset to bridge</message>
+  <message name="IDS_BRAVE_WALLET_BRIDGING_VIA" desc="Bridging via provider label">Bridging via</message>
+  <message name="IDS_BRAVE_WALLET_SWAPPING_VIA" desc="Bridging via provider label">Swapping via</message>
+  <message name="IDS_BRAVE_WALLET_REVIEW_SWAP" desc="Review swap button">Review swap</message>
+  <message name="IDS_BRAVE_WALLET_REVIEW_BRIDGE" desc="Review bridge button">Review bridge</message>
+  <message name="IDS_BRAVE_WALLET_EST_FEES" desc="Bridge and Swap estimated fees label">Est fees</message>
+  <message name="IDS_BRAVE_WALLET_EST_TIME" desc="Bridge and Swap estimated time label">Est time</message>
+  <message name="IDS_BRAVE_WALLET_EXCHANGE_RATE" desc="Bridge and Swap exchange rate label">Exchange rate</message>
+  <message name="IDS_BRAVE_WALLET_EXCHANGE_FOR" desc="Bridge and Swap exchange rate value for value label"><ph name="ONE_TOKEN_VALUE"><ex>1 BAT</ex>$1</ph> for <ph name="EXCHANGE_VALUE"><ex>5 USDC</ex>$2</ph></message>
+  <message name="IDS_BRAVE_WALLET_PROVIDER_API" desc="Bridge and Swap provider API name"><ph name="PROVIDER_NAME"><ex>Ox</ex>$1</ph> API</message>
```

Comment:
Let's get rid of this translation and just use 0x, LiFi, Jupiter, etc as the provider name.

Reply:
Done! https://github.com/brave/brave-core/compare/5412f4cbdc0c90ffea513cccbbba4568ea85e16a..398011807556d29c6eb1ab1a71f1e2a9287e8edd

![Screenshot 12](https://github.com/brave/brave-core/assets/40611140/b609862a-5b82-47e3-92dd-126c18a24539)